docs/run-taco.md

```diff
@@ -46,4 +46,4 @@ Signature verification was added to Tableau Desktop in 2019.4 and Tableau Server
 
 To use an unsigned `.taco` file, you can disable signature verification using the following command line argument: `-DDisableVerifyConnectorPluginSignature=true`.
```

Comment:
Do we want to specify this is for Desktop ? As in the para below this we mention server explicitly